CHANGE_LOG

```diff
@@ -1,3 +1,92 @@
+Version 0.39.0
+--------------
+
+Here are the highlights for the Numba 0.39.0 release.
+
+* This is the first version that supports Python 3.7.
+* With help from Intel, we have added fixed the SVML support (related issues
+  #2938, #2998, #3006).
+* List has gained support for containing reference-counted types like NumPy
+  arrays and `list`.  Note, list still can't hold heterogeneous types.
+* We have done a significant change to the internal calling-convention,
+  which should be transparent to most users, to allow for a future feature to
```

Comment:
s/to allow for a future feature to jump/to allow for a future feature that will permit jumping/ ?